cves/kernel/CVE-2013-3224.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The bt_sock_recvmsg function in the Linux kernel did not properly set the
```

Comment:
I would make this description a little more general, eliminating the need for a function name.

Reply:
thanks. I updated it now.